### PR TITLE
Datastore csv related scripts

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

**Comment by fanjinfei** [comment]
ready for review

File: bin/rebuild_pd_solr_from_uploaded_csv.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#/var/www/html/open_gov/open_data/ckanext-canada/bin/csv2solr.sh \
+#    /var/www/html/open_gov/open_data/ckan/production-cli.ini \
+#    http://f7odwebb4.stcpaz.statcan.gc.ca/vl \
+#    contracts:contracts \
+#    contractsa:contractsa \
+#    grants:grants \
+#    hospitalityq:hospitalityq \
+#    reclassification:reclassification \
+#    travelq:travelq \
+#    wrongdoing:wrongdoing
```
Comment (wardi) [inline bin/rebuild_pd_solr_from_uploaded_csv.sh]: please remove commented-out code and "new-way" above

File: bin/upload_ati_csv_from_datastore_tables.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+#mv "${TMPDIR}/hospitalityq.csv" "${TMPDIR}/raw-hospitalityq.csv"
+#bin/filter_disclosure_group.py \
+#        < "${TMPDIR}/raw-hospitalityq.csv" \
+#        > "${TMPDIR}/hospitalityq.csv"
+#mv "${TMPDIR}/travelq.csv" "${TMPDIR}/raw-travelq.csv"
+#bin/filter_disclosure_group.py \
+#        < "${TMPDIR}/raw-travelq.csv" \
+#        > "${TMPDIR}/travelq.csv"
+
+# NEW WAY TO UPLOAD
```
Comment (wardi) [inline bin/upload_ati_csv_from_datastore_tables.sh]: remove comments here too

**Comment by wardi** [comment]
let's remove "pd" and "ati" from the script names because they cover more than just those now.

File: bin/upload_csv_from_datastore_tables.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+cp inventory.csv /var/www/html/static/inventory.csv
+/var/www/html/rc_reg/ckanext-canada/bin/csv2xlsx.py \
+	< inventory.csv > /var/www/html/static/inventory.xlsx
+
+# Custom business logic (filtering out records) here
+cd /var/www/html/rc_reg/ckanext-canada
+mv "${TMPDIR}/contracts.csv" "${TMPDIR}/raw-contracts.csv"
+bin/contracts_10k_filter.py \
+        < "${TMPDIR}/raw-contracts.csv" \
+        > "${TMPDIR}/contracts.csv"
```
Comment (wardi) [inline bin/upload_csv_from_datastore_tables.sh]: @fanjinfei I mean remove these 4 lines and the contracts_10k_filter.py script

Comment (fanjinfei) [inline bin/upload_csv_from_datastore_tables.sh]: sorry. I should ask first.

**Pull request merged**